modules/ROOT/examples/live-demos/ai-demo/index.js

```diff
@@ -0,0 +1,13 @@
+// a generic TinyMCE configuration with ai plugin and toolbar added.
+
+tinymce.init({
+  selector: 'textarea#ai',
+  plugins: [
+    "advlist", "anchor", "autolink", "charmap", "code", "fullscreen", 
+    "help", "image", "insertdatetime", "link", "lists", "media", 
+    "preview", "searchreplace", "table", "visualblocks", "ai",
+    ],
+  height: 1200,
+  toolbar: "aidialog | undo redo |link image accordion | styles | bold italic underline strikethrough | align | bullist numlist"
+});
+
```

Comment:
I assume this is just a placeholder? Because this demo will not be a valid implementation. (missing `ai_request`)

Reply:
> I assume this is just a placeholder?

You assume correctly. We’re not including a live-demo in this iteration of the AI Assistant documentation.

And the two files in /live-demos/ai-demo/ are just placeholders for when (and if) we do add one later.